courses/models.py

```diff
@@ -530,41 +530,6 @@ def __str__(self):
         title = f"{self.courseware_id} | {self.title}"
         return title if len(title) <= 100 else title[:97] + "..."
 
-    def clean(self):
-        """
-        If expiration_date is not set:
-        1. If end_date is provided: set expiration_date to default end_date + 90 days.
-        2. If end_date is None, don't do anything.
-
-        Validate that the expiration date is:
-        1. Later than end_date if end_date is set
-        2. Later than start_date if start_date is set
-        """
-        if not self.expiration_date:
-            if self.end_date:
-                self.expiration_date = self.end_date + timedelta(days=90)
-            return
-
-        if self.start_date and self.expiration_date < self.start_date:
-            raise ValidationError("Expiration date must be later than start date.")
-
-        if self.end_date and self.expiration_date < self.end_date:
-            raise ValidationError("Expiration date must be later than end date.")
```

Comment:
from the UI prospective, how can we ignore these validations ?

Reply:
I don't think we should in the first place. The validation is to make sure the entered data is correct, and if we start ignoring that it defeats the whole purpose of the validation. This is primarily going to be done via the Django admin and it works as expected.